packages/client/tests/functional/jest.config.js

```diff
@@ -33,7 +33,10 @@ module.exports = () => {
         addFileAttribute: 'true',
         ancestorSeparator: ' › ',
         classNameTemplate: (vars) => {
-          return vars.classname.replace(/ \(provider=.*?\)/g, '')
+          return vars.classname
```

Comment:
These replaced values prevent BuildPulse to group flaky tests by name, so we remove that info, which will be available in the errors that BuildPulse has collected.